packages/third-parties/bullmq/README.md

```
@@ -255,6 +257,30 @@ class MyService {
 }
 ```
 
+## Logs
+
+Be default the module will log every job's start and completion/failure.
+
+Start and completion is logged on the level of `info`, failure is logged on the level of `error`.
+To change/disable this behaviour, pass `logLevel` in the configuration 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)